ivy_tests/test_ivy/test_frontends/test_torch/test_random_sampling.py

```diff
@@ -0,0 +1,41 @@
+# global
+from hypothesis import given
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+@handle_cmd_line_args
+@given(
+    population_size=helpers.ints(),
```

Comment:
Should be a tensor. Look at some other tests and the helper functions.

Reply:
I am a bit confused about that because I couldn't find a similar function for tensors in helper functions or other tests. Can you please provide more guidance? Thanks a lot for your reviews